Hearthstone Deck Tracker/Hearthstone/GameV2.cs

```diff
@@ -78,6 +78,7 @@ public bool IsOpponentMinionInPlay
 		public int OpponentMinionCount => Entities.Count(x => (x.Value.IsInPlay && x.Value.IsMinion && x.Value.IsControlledBy(Opponent.Id)));
 		public int PlayerMinionCount => Entities.Count(x => (x.Value.IsInPlay && x.Value.IsMinion && x.Value.IsControlledBy(Player.Id)));
 		public int OpponentHandCount => Entities.Count(x => x.Value.IsInHand && x.Value.IsControlledBy(Opponent.Id));
+		public int OpponentSecretCount => Entities.Count(x => x.Value.IsSecret && x.Value.IsControlledBy(Opponent.Id));
```

Comment:
This will need to check `x.Value.IsInSecret`, otherwise you get secrets from the graveyard.